### PR TITLE
Rename `FileTree` to `CacheManager`

### PR DESCRIPTION
**Pull request by riya-17**
PBENCH-947

This PR deals with renameing FileTree and it's related objects to CacheManager.

### CODE REVIEW

File: lib/pbench/test/unit/server/test_cache_manager.py [hunk 1]
```diff
@@ -68,12 +68,12 @@ def selinux_enabled(monkeypatch):
...
 
-class TestFileTree:
+class TestCacheManager:
     def test_create(self, server_config, make_logger):
         """
-        Create an empty FileTree object and check the properties
+        Create an empty CacheManager object and check the properties
         """
-        tree = FileTree(server_config, make_logger)
+        tree = CacheManager(server_config, make_logger)
```
Comment (dbutenhof) [inline lib/pbench/test/unit/server/test_cache_manager.py:76]: It's interesting that you chose to rename all the local `FileTree` instances to `cache_m` in the product code but left `tree` in unit tests.

I'm not thrilled about `cache_m` (every time I see it, it seems "unfinished"), and we don't need to have total consistency between product code and unit tests, but it doesn't hurt either, and `tree = CacheManager(...)` doesn't really make much sense.

I would go with `cache` (which is admittedly overly generic) or maybe `cacheman` (which is more specific but/and also a bit silly)... but at least I'd like to advocate (lightly) for consistency and at least a better name on the testing side.

**Review by dbutenhof — APPROVED** [review]
Despite one comment I think would slightly improve the readability, I have no technical issues and I thank you for separating this trivial mass change out into a separate PR. 👍🏻 

If you don't feel like making the change or subsequent reviews don't give another reason to make changes, I'll resolve the conversation.

**Comment by dbutenhof** [comment]
Ah; your module rename caused `isort` to complain:

```
ERROR: /home/jenkins/pbench/lib/pbench/server/api/resources/upload_api.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/jenkins/pbench/lib/pbench/server/api/resources/query_apis/datasets_delete.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/jenkins/pbench/lib/pbench/test/unit/server/test_cache_manager.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/jenkins/pbench/lib/pbench/test/unit/server/test_datasets_inventory.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/jenkins/pbench/lib/pbench/test/unit/server/test_requests.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/jenkins/pbench/lib/pbench/test/unit/server/query_apis/test_datasets_delete.py Imports are incorrectly sorted and/or formatted.
```

File: lib/pbench/test/unit/server/test_cache_manager.py (hunk 1 above)
Comment (webbnh) [reply lib/pbench/test/unit/server/test_cache_manager.py:76]: For the most part, the references to `cache_m` are localized enough that we probably don't care much what they are named.

However, I'm with Dave that sticking with `tree` here seems a bit...anachronistic?  🙂

`cache` is OK with me.  (While `cacheman` would be OK, too, but, for a scope of just half a screen, even `cm` would be fine....)

**Review by webbnh — APPROVED** [review]
Like Dave said, thank you for making this change in a stand-alone PR!  I, too, have no issue other than the one that he identified, which will go away when he resolves his conversation.  🙂

**Review by npalaska — DISMISSED** [review]

**Review by siddardh-ra — APPROVED** [review]
Looks good

**Review by dbutenhof — APPROVED** [review]
👍🏻

**Pull request merged**